### PR TITLE
[Flang] make ppc unsupported for x86_64 test case (NFC)

### PR DESCRIPTION
**Pull request by madanial0**
The test case is for x86_64 adding powerpc as unsupported.

### CODE REVIEW

File: flang/test/Semantics/kinds04_q10.f90 [hunk 1]
```diff
@@ -7,7 +7,7 @@
 !
 ! This test is for x86_64, where exponent-letter 'q' is for
 ! 10-byte extended precision
-! UNSUPPORTED: system-windows
+! UNSUPPORTED: system-windows, target=powerpc{{.*}}
```
Comment (DanielCChen) [inline flang/test/Semantics/kinds04_q10.f90:10]: This is rather a question. 
Should the test case specify REQUIRED platform instead of UNSUPPORTED?

Comment (madanial0) [reply flang/test/Semantics/kinds04_q10.f90:10]: it seems that previously it was excluded using unsupported not sure if we should exclude it from other platforms by using required

File: flang/test/Semantics/kinds04_q10.f90 [hunk 2]
```diff
@@ -7,7 +7,7 @@
 !
 ! This test is for x86_64, where exponent-letter 'q' is for
 ! 10-byte extended precision
-! UNSUPPORTED: system-windows
+! UNSUPPORTED: system-windows, system-aix
```
Comment (kkwli) [inline flang/test/Semantics/kinds04_q10.f90:10]: Why not `target=powerpc{{.*}}` as PPC is non-x86_64 regardless of the OS?

Comment (kkwli) [reply flang/test/Semantics/kinds04_q10.f90:10]: Ok

**Review by kkwli — APPROVED** [review]
LG.  Thanks

**Pull request merged**